### ISSUE TITLE
DropDownMenu onOptionSelected not responding

### ISSUE THREAD

**Issue by ekmutai** [shoutem/ui#301]
onOptionSelected doesn't seem to be working even on i can't switch from the typography option, only close button works.
am using "@shoutem/ui": "^0.21.2" , and i set up the project using create-react-app
![simulator screen shot 15 aug 2017 17 32 20](https://user-images.githubusercontent.com/2639438/29327987-a761ad94-81f9-11e7-84ca-0224f8bd7e33.png)

**Comment by Vladimir-Vdovic** [comment]
I've done my best to replicate it, but the drop down menus work fine for me throughout all the example categories.

What I did:

`react-native init DropDownTest`
`cd DropDownTest`
`npm install @shoutem/ui --save`
`react-native link`

Replaced Hello World bootstrap in `index.ios.js` with:
```
import React, { Component } from 'react';
import { AppRegistry } from 'react-native';
import { Examples } from '@shoutem/ui';

class DropDownTest extends Component {
  render() {
    return (
      <Examples />
    );
  }
}

AppRegistry.registerComponent('DropDownTest', () => DropDownTest);
```

`react-native run-ios`

The drop down menus function as expected, selections work, closing works, states update in examples where the selection changes state (e.g. Dropdown Large example).

Can you try that again and see how it works please?

**Comment by Morriz** [comment]
Does it have anything to do with Expo's examples' deferred loading of assets? Maybe it's not linking any of the menu's items?

**Comment by Morriz** [comment]
I suggest you look at the file `ui/examples/create-react-native-app/App.js`. I presume it is under your care. It is there that the deferred rendering takes place.

**Comment by Vladimir-Vdovic** [comment]
I was able to reproduce this using `create-react-native-app` and using our default example. We'll look into what edits we may need to make to the `App.js` example file in order to get it working, or if there's something else prohibiting the UI toolkit from being used with CRNA.

There were some other issues with CRNA, visible [here](https://github.com/shoutem/ui/issues/262), but I was unable to reproduce them, that thread may have some relevant information as well.

**Comment by brandon-dewitt** [comment]
I am using RN 0.42.0 and tried upgrading @shoutem/ui from 0.14.3 to 0.21.3.  After upgrading onOptionSelected is not called.

**Comment by handiwijoyo** [comment]
Is there any fix for this yet? I have this issue as well.

**Comment by Vladimir-Vdovic** [comment]
Hey guys,

We've just released [version 0.22.0](https://www.npmjs.com/package/@shoutem/ui) of the UI toolkit, we've addressed this issue in them (among a few others). Shoutem's UI toolkit is now also compatible with React Native 0.50.3.

Closing issue as it's resolved.
